### ISSUE TITLE
nan issues.

### ISSUE THREAD

**Issue by DarkAlchy** [TheLastBen/fast-stable-diffusion#2608]
I have to turn the LR down to -8 (3e-8) or I will get a nan.  1e-6 is a nan in 3 epochs using 40 images.  Any idea why this would be or a way to slow the learning down (or even better why my LR has to go so low when 1e-6 works for others)?  This is for a SDXL LoRA DB.

**Comment by TheLastBen** [comment]
it's actually a good indicator of overfitting, loss going to nan means that your lr settings are too high

**Comment by DarkAlchy** [comment]
Why?  Why am I being forced to use -8 or -9 or I get a nan like that.  I know what it means but the reason I asked is WHY is it happening, and only with yours with the same dataset.

**Comment by TheLastBen** [comment]
the loss goes nan when the model learns the concept too fast, some concept converge faster than others, so using a lower learning rate will help you get better and more stable results
